CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeEditorPanel.java

```diff
@@ -837,6 +837,7 @@ private void editKeysAction(final GraphElementType elementType) {
             final DefaultGetter<List<String>> defaultGetter = new PrimaryKeyDefaultGetter(elementType);
             final AbstractEditor<List<String>> editor = LIST_SELECTION_EDITOR_FACTORY.createEditor(editOperation, defaultGetter, String.format("Edit primary key for %ss", elementType.getShortLabel()), currentKeyAttributes);
             ((ListSelectionEditor) editor).setPossibleItems(allAttributes);
+            editor.untickEditCB();
```

Comment:
I think the creation of this function isn't actually necessary. A quick look at the classes you removed the `noValueCheckbox` (moved upstream) suggests that they are setting this value in their own code on loading the dialog. They also do this before your function call which means you're simply changing whatever they set it to. I think it might be better to change the set at that original point instead (I would check out the `updateControlsWithValue` function that has been overriden in each of those factory classes)